system_metrics_collector/src/system_metrics_collector/constants.hpp

```diff
@@ -25,10 +26,11 @@ namespace collector_node_constants
 constexpr const char kStatisticsTopicName[] = "system_metrics";
 
 constexpr const char kCollectPeriodParam[] = "measurement_period";
-constexpr const int64_t kDefaultCollectPeriodInMs = 1000;   // 1 second
+constexpr const std::chrono::milliseconds kDefaultCollectPeriod{1000};    // 1 second
```

Comment:
Why is this still in milliseconds when the delay is now 1 second?

Reply:
I need to get the `count()` in milliseconds at some point.